packages/hardcaml-reedsolomon/hardcaml-reedsolomon.0.2/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+maintainer: "andy.ray@ujamjar.com"
+homepage: "https://github.com/ujamjar/hardcaml-reedsolomon"
+build: [
+  [make "all"]
+  [make "install"]
```

Comment:
if you run `opam lint` on this, it will warn you that you can have a `install:` field now. Not a blocker for merge though.